pkg/polaris/polaris_test.go

```diff
@@ -105,7 +105,7 @@ func TestAwsAccountAddAndRemove(t *testing.T) {
 	// Add the default AWS account to Polaris. Usually resolved using the
 	// environment variables AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and
 	// AWS_DEFAULT_REGION.
-	id, err := client.AWS().AddAccount(ctx, aws.Default(), core.FeatureCloudNativeProtection,
+	id, err := client.AWS().AddAccount(ctx, aws.Profile(testAccount.Profile), core.FeatureCloudNativeProtection,
 		aws.Name(testAccount.AccountName), aws.Regions("us-east-2"))
```

Comment:
Shouldn't the region here be picked up from the profile (as specified by the `config` file)?

Reply:
Also applies to the check at line 129 (which isn't comment-able since it is.. too far from a change?!)

Reply:
No this is the region that the customer wants to protect. The default region is required by the AWS SDK (it's used by the STS caller id call) and it's also where we create the CloudFormation.